soh/soh/Enhancements/mods.cpp

```diff
@@ -1018,6 +1018,31 @@ void RegisterRandomizerSheikSpawn() {
     });
 }
 
+void UpdateHurtContainerModeState(bool newState) {
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::OnPlayerUpdate>([newState]() {
+        bool hurtEnabled;
```

Comment:
```suggestion
        static bool hurtEnabled = false;
```

otherwise it won't remember the state

Reply:
oh, but you still have it `OnPlayerUpdate`...

Reply:
my suggestion was based on mirror mode

https://github.com/HarbourMasters/Shipwright/blob/dd6271ecae9868c99bdc79324c6143f082fbcc65/soh/soh/SohMenuBar.cpp#L1108-L1110

https://github.com/HarbourMasters/Shipwright/blob/dd6271ecae9868c99bdc79324c6143f082fbcc65/soh/soh/Enhancements/mods.cpp#L636-L640

that one is registered on scene init (assuming because it needs to happen on transition), but also directly called when the cvar is toggled. i figure for this it just needs to happen when loading a file, then any containers/pieces will be handled appropriately with the `if(cvar)` stuff in game